sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1867,13 +1867,93 @@ def _df_getitem_unicode_idx_impl(self, idx):
     ty_checker.raise_exc(idx, expected_types, 'idx')
 
 
+def df_getitem_single_label_loc_codegen(self, idx):
+    """
+    Example of generated implementation:
+        def _df_getitem_single_label_loc_impl(self, idx):
+            idx_list = []
+            for i in range(len(self._dataframe.index)):
+                if self._dataframe._index[i] == idx:
+                idx_list.append(i)
+            data_0 = []
+            for i in numba.prange(len(idx_list)):
+                index_in_list_0 = idx_list[i]
+                data_0.append(self._dataframe._data[0][index_in_list_0])
```

Comment:
you can't do `append` in `prange` loop. Also you could use `sdc_take`. @kozlov-alexey